### DISCUSSION TITLE
Appointment add external person (not in address book) possible?

### DISCUSSION THREAD

**Discussion by slu-010101** [Intermesh/groupoffice#1110] [Q&A] [answered]
Hi,

is it possible to add/invite external person in an appointment?
I find only the option to select a user or a person from the address book.

I'm new at Group Office, possible to overlook a option/function.

**Comment by slu-010101** [comment]
@mschering is there a support ticket (to buy) to clarify these questions?
Group Office is a very interesting project.

**Comment by mschering** [comment, marked as answer]
Thanks, we offer premium support when you have the pro license or our cloud service.

To answer your question, at the moment the contact has to be in the address book. We are near to complete a new calendar module where you can enter any e-mail address.

**Reply by slu-010101** [reply]
This sounds very good, thank you.

**Comment by lonesomewalker** [comment]
Does this also apply to ressources?
